internal/api/service/service.go

```diff
@@ -18,6 +19,7 @@ type Config struct {
 	L2RpcEndpoint  string
 	L1PollInterval int
 	L2PollInterval int
+	SlackURL       string
```

Comment:
Do we intend to do anything with this value? We may just wanna rm it for the time being